docs/content/en/themes-docs.md

```diff
@@ -210,6 +210,9 @@ You can create a `content/settings.json` file to configure the theme:
 - `github`
   - Type: `String`
   - *The GitHub repository of your project `${org}/${name}` to display the last version, the releases page, the link at the top and the `Edit this page on GitHub link` on each page*
+- `defaultBranch`
```

Comment:
@danielroe Could you add the default value to `main` here?

Reply:
:+1: Done.